lib/packwerk/cli.rb

```diff
@@ -177,6 +185,11 @@ def check(paths)
       all_offenses.empty?
     end
 
+    sig { returns(FileProcessor) }
+    def file_processor
+      @file_processor ||= FileProcessor.new(node_processor_builder: @node_processor_builder)
```

Comment:
The CLI already knows way too much and I don't think it should know about `FileProcessor`.

Reply:
Ideally the CLI would just be a thin layer that translates one command line call into one to three ruby method calls.